jerry-core/ecma/builtin-objects/ecma-builtin-array.inc.h

```diff
@@ -32,10 +32,17 @@ OBJECT_VALUE (LIT_MAGIC_STRING_PROTOTYPE,
 /* Number properties:
  *  (property name, object pointer getter) */
 
+#if ENABLED (JERRY_ES2015)
+/* ECMA-262 v6, 22.1.2 */
+NUMBER_VALUE (LIT_MAGIC_STRING_LENGTH,
+              1,
+              ECMA_PROPERTY_FLAG_CONFIGURABLE)
```

Comment:
Perhaps we could introduce a macro which is different for ES5.1 and 6.0. That would reduce the use of guards.

`ECMA_PROPERTY_FLAG_DEFAULT_LENGTH` or something

Reply:
I replaced the macro as you said. Thank you for the review.